src/ts/main.ts

```diff
@@ -17,4 +19,11 @@ var mainView = myApp.addView('.view-main', {
 
 var mySwiper = myApp.swiper('.swiper-container', {
     pagination:'.swiper-pagination'
-});
\ No newline at end of file
+});
+
+function parseSendClick() {
+	var toAddr = ethUtil.stripHexPrefix($$("#to-address").val());
+	var amount = $$("#eur-amount").val()*100;
+	console.log("Sending amount " + amount + " to " + toAddr );
+	KE_instance.eth.sendDelegatedTransfer(KE_instance.ethAddress,toAddr,amount,KE_instance.eth.gatewayFees,3)
```

Comment:
as i understand whole idea of KE_instance is to have access on 'eth', i think it can be done easier with adding parseSendClick to window from Kryptoeuro instance, then you will no need in KE_instance